server/application/application.proto

```diff
@@ -180,139 +182,139 @@ message ManagedResourcesResponse {
 // ApplicationService
 service ApplicationService {
 
-	// List returns list of applications
```

Comment:
Looks like files was reformatted to use spaces instead of tabs. Can you please keep tabs?

Reply:
sure, I did not do it intentionally. I will revert.

Reply:
Done